lerna.json

```diff
@@ -1,8 +1,10 @@
 {
   "version": "independent",
-  "lerna": "2.0.0-beta.38",
+  "lerna": "2.0.0",
+  "npmClient": "yarn",
   "packages": [
     "packages/*"
   ],
-  "npmClient": "yarn"
+  "registry": "https://registry.npmjs.org/",
```

Comment:
This fixes a bug with publishing in the official Lerna 2 release when using yarn by telling it to publish to the npm registry, not the yarn registry.